models/ios/lag_interfaces/override_example_01.txt

```diff
@@ -0,0 +1,56 @@
+# Using overridden
+
+# Before state:
+# -------------
+#
+# vios#show running-config | section ^interface
+# interface Port-channel10
+#  flowcontrol receive on
+# interface Port-channel20
+# interface Port-channel30
+# interface GigabitEthernet0/1
+#  shutdown
+#  flowcontrol receive on
+#  channel-group 10 mode auto
+# interface GigabitEthernet0/2
+#  shutdown
+#  flowcontrol receive on
+#  channel-group 10 mode auto
+# interface GigabitEthernet0/3
+#  shutdown
+#  channel-group 20 mode on
+# interface GigabitEthernet0/4
+#  shutdown
+#  channel-group 30 mode active
+
+- name: Override device configuration of all interfaces with provided configuration
+  ios_interfaces:
+    config:
+      - name: 20
+        mode: auto
+        members:
+        - GigabitEthernet0/2
+        - GigabitEthernet0/2
```

Comment:
@justjais something is not right with this example.  Do you intend to use GigabitEthernet0/2 in this example twice?  I think it should be GigabitEthernet0/3